Lib/test/test_pdb.py

```diff
@@ -1771,6 +1771,18 @@ def test_errors_in_command(self):
             '(Pdb) ',
         ])
 
+    def test_issue34266(self):
+        '''do_run handles exceptions from parsing its arg'''
+        commands = "\n".join([
+            'run \\',
```

Comment:
It is worth to add also a test for `run "`. There are only to causes of ValueError in `shlex.split()`. Interesting, that gdb handles `r \` and `r "` in different ways, so it is possible that we will differentiate them too in future.